client/cypress/e2e/_13metrics.cy.ts

```diff
@@ -0,0 +1,192 @@
+        cy.get('[data-test=MetricsEpochHeader__NavigationArrows]').should('be.visible');
+        cy.get('[data-test=MetricsEpochHeader__NavigationArrows__leftArrow]').click();
+        cy.get('[data-test=MetricsEpochHeader__title]')
+          .invoke('text')
+          .should('eq', isMobile || isTablet ? 'E2 Allocation' : 'Epoch 2 Allocation Window');
+        cy.get('[data-test=MetricsEpochHeader__NavigationArrows__rightArrow]').click();
+        cy.get('[data-test=MetricsEpochHeader__title]')
+          .invoke('text')
+          .should('eq', isMobile || isTablet ? 'E3 Allocation' : 'Epoch 3 Allocation Window');
+      });
+
+      it('"Epoch" section title has epoch dates range when epoch is past', () => {
+        cy.get('[data-test=MetricsEpochHeader__openBadge]').should('be.visible');
+        cy.get('[data-test=MetricsEpochHeader__epochDuration]').should('not.exist');
+        cy.get('[data-test=MetricsEpochHeader__NavigationArrows__leftArrow]').click();
+        cy.get('[data-test=MetricsEpochHeader__openBadge]').should('not.exist');
+        cy.get('[data-test=MetricsEpochHeader__epochDuration]').should('be.visible');
+      });
+
+      it('Next arrow button is disabled when there is no more next epochs', () => {
+        cy.get('[data-test=MetricsEpochHeader__NavigationArrows__rightArrow]')
+          .then($el => $el.css('backgroundColor'))
+          .should('be.colored', '#fafafa');
+        cy.get('[data-test=MetricsEpochHeader__NavigationArrows__rightArrowSvg] path')
+          .then($el => $el.css('fill'))
+          .should('be.colored', '#ebebeb');
+        cy.get('[data-test=MetricsEpochHeader__title]')
+          .invoke('text')
+          .should('eq', isMobile || isTablet ? 'E3 Allocation' : 'Epoch 3 Allocation Window');
+        cy.get('[data-test=MetricsEpochHeader__NavigationArrows__rightArrow]').click({
+          force: true,
+        });
+        cy.get('[data-test=MetricsEpochHeader__title]')
+          .invoke('text')
+          .should('eq', isMobile || isTablet ? 'E3 Allocation' : 'Epoch 3 Allocation Window');
+      });
+
+      it('Prev arrow button is disabled when there is no more prev epochs ', () => {
+        cy.get('[data-test=MetricsEpochHeader__NavigationArrows__leftArrow]')
+          .then($el => $el.css('backgroundColor'))
+          .should('be.colored', '#fafafa');
+        cy.get('[data-test=MetricsEpochHeader__NavigationArrows__leftArrowSvg] path')
+          .then($el => $el.css('fill'))
+          .should('be.colored', '#9ea39e');
+        cy.get('[data-test=MetricsEpochHeader__title]')
+          .invoke('text')
+          .should('eq', isMobile || isTablet ? 'E3 Allocation' : 'Epoch 3 Allocation Window');
+        cy.get('[data-test=MetricsEpochHeader__NavigationArrows__leftArrow]').click({
+          force: true,
+        });
+        cy.wait(1000);
+        cy.get('[data-test=MetricsEpochHeader__NavigationArrows__leftArrow]').click({
+          force: true,
```

Comment:
Why? `force` means it's not actually clickable for the user, indicating a problem with UI.

Reply:
Fixed